secondary.js

```diff
@@ -1,26 +1,65 @@
+const studentIdNum = Math.floor(Math.random() * 100000000) +1;
```

Comment:
This could be converted into a function. I see it being used in multiple places with the same code over and over again. Also, though it's not probable, but this could result in some items having the same id. Especially if this were to have 100000000 or more students, which is possible.